core/vulkan/vk_virtual_swapchain/cc/virtual_swapchain_android.exports

```diff
@@ -4,3 +4,4 @@ vkEnumerateInstanceLayerProperties
 vkEnumerateInstanceExtensionProperties
 vkEnumerateDeviceLayerProperties
 vkEnumerateDeviceExtensionProperties
+virtual_swapchain_write_png
```

Comment:
I don't think we should be passing this out directly. This bypasses the vulkan layer system. 

We should do this the same way as we set the callback. This callback is done by passing it into the vulkan layer.